flowmachine/flowmachine/features/location/redacted_labelled_aggregate.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import List, Union
+
+from flowmachine.features.location.labelled_flows import LabelledFlows
+from flowmachine.features.location.labelled_spatial_aggregate import (
+    LabelledSpatialAggregate,
+)
+from flowmachine.core import Query
+
+
+class RedactedLabelledAggregate(Query):
+    """
+    Query that drops any locations aggregates or flows that, when disaggregated by label, reveal a number of subscribers
+    less than redaction_threshold
+    Parameters
+    ----------
+    labelled_query: LabelledSpatialAggregate or LabelledFlows
+        The LabelledSpatialAggregate query to redact
+    redaction_threshold: int default 15
+        If any labels within a location reveal less than this number of subscribers, that location is dropped
```

Comment:
```suggestion
        If any labels within a location reveal this number of subscribers or fewer, that location is dropped
```
Changing wording here to reflect that the threshold is `min(value) > redaction_threshold`, not `min(value) >= redaction_threshold`